src/network/http/websocket.cpp

```diff
@@ -300,6 +300,10 @@ namespace fc { namespace http {
                   _server.close( item.first, 0, "server exit" );
 
                if( _closed ) _closed->wait();
+#ifdef TRAVIS_BUILD
```

Comment:
I thought abit had fixed this somewhere else?
Is it still required? If it is should be in a separate PR.

Reply:
Not sure, but I'll remove it.

Reply:
iirc it was a hack and we never included it. not against adding it.